utils/node_local_test.py

```diff
@@ -1031,6 +1031,15 @@ def run_il_test(server, conf):
     # Read it from within a container
     ret = il_cmd(dfuse, ['md5sum', os.path.join(dirs[-1], 'bash')])
     assert ret.returncode == 0
+    ret = subprocess.run(['dd',
+                          'if={}'.format(os.path.join(dirs[-1], 'bash')),
+                          'of={}'.format(os.path.join(dirs[-1], 'bash_copy')),
+                          'iflag=direct',
+                          'oflag=direct',
+                          'bs=128k'])
+
+    print(ret)
```

Comment:
is this a debug print left in?

Reply:
ret contains the return code and the stdout so it's useful to print here, this was intentional.